src/query/mod.rs

```diff
@@ -48,6 +55,43 @@ impl fmt::Display for Timestamp {
     }
 }
 
+#[cfg(any(test, feature = "chrono_timestamps"))]
+impl Into<DateTime<Utc>> for Timestamp {
+    fn into(self) -> DateTime<Utc> {
+        match self {
+            Timestamp::Now => Utc::now(),
+            Timestamp::Hours(h) => {
+                let millis = h * 60 * 60 * 1000;
```

Comment:
In the tests you are using `MINUTES_PER_HOUR` and the like, wouldn't it make sense to do the same here? Or maybe we can even use `chrono::Duration` or similar for this?